topics/persistence.md

```diff
@@ -68,6 +68,17 @@ dataset to disk every 60 seconds if at least 1000 keys changed:
 
 This strategy is known as _snapshotting_.
 
+## No persistence
+
+If you prefer **not** to have persistence (eg. when using a Valkey instance solely as a cache) that is also a possibility.
+
+First comment out or remove all existing `save` lines that are present in the current configuration file.   
+Then add the following configuration instead:
```

Comment:
This sort of assumes there's a lot of junk in the user's config file. I'd prefer to focus first on the correct config and only afterwards mention what to do with extra junk in the files, if any. How about this?

```suggestion
RDB snapshotting is enabled by default. To disable it, set the `save` configuration parameter to the empty string `""` and remove any excess `save` lines that are present in the current configuration file.
```